venom.go

```diff
@@ -216,13 +215,19 @@ func (v *Venom) getUserExecutorFilesPath(vars map[string]string) (filePaths []st
 }
 
 func (v *Venom) registerUserExecutors(ctx context.Context, name string, vars map[string]string) error {
+	_, ok := v.executorsUser[name]
```

Comment:
We can't do a cache here. The GetExecutorRunner is used in RunUserExecutor, so that `vars` can contains different value depending of the input.

Reply:
Will mark it as WIP as I need to check if I have migrated all the necessary changes for it to work properly. 

You do need to check if you have read the executor or not as this is the point of the cache , we want to read that file only once and not multiple times ) .